### ISSUE TITLE
Lack of Nonce management in the client

### ISSUE THREAD

**Issue by botnumberseven** [OriginTrail/dkg.py#12]
Asset.create function lacks an option to manage Nonce, which limits number of assets which can be published from one wallet. Essentially current implementation allows only sequential publishing.
I can not start publishing next asset while previous asset publishing process hasn't finished yet. That's because next asset pub transaction goes with the same Nonce as previous asset pub transaction.

Functional workaround here - use different wallets if there is a need to publish more then ~50 assets per hour.
But it would be nice to have an ability to publish several assets in parallel from the same wallet.

**Comment by u-hubar** [comment]
@botnumberseven, unfortunately this is an EVM limitation. Even if there is a possibility to control nonce manually, transactions are still mined in sequence (ordered by nonce) on-chain. So, even if you're submitting multiple transactions in parallel, they still will be mined in sequence and if one of the transactions fails, it will block all other submitted transactions with bigger nonces.

Using multiple wallets seems the best available option to me.
